contracts/interchain-token-service/tests/deploy_interchain_token.rs

```diff
@@ -32,10 +32,10 @@ fn deploy_interchain_token_with_initial_supply_no_minter() {
     let token_meta_data = setup_token_metadata(&env, "name", "symbol", 6);
     let initial_supply = 100;
 
-    let (deployed_address, _token_id) =
+    let token_id =
         client.deploy_interchain_token(&sender, &salt, &token_meta_data, &initial_supply, &minter);
-
-    let token = InterchainTokenClient::new(&env, &deployed_address);
+    let token_data = client.token_data(&token_id);
+    let token = InterchainTokenClient::new(&env, &token_data.token_address);
```

Comment:
test that token manager type was set correctly as well for the positive tests

Reply:
updated